apps/web/src/actions/copilot/refinePrompt.ts

```diff
@@ -116,5 +118,10 @@ export const refinePromptAction = authProcedure
       },
     })
 
-    return result.response.text
+    return (
```

Comment:
Instead of lying here you can really throw an error if it's not `object`

```javascript
if (result.response.type !== 'object') {
  throw new Error('This should never happen')
}

// Now Typescript is happy and you can do `result.response.object`
```